### PR TITLE
React eslint

### PR DESCRIPTION
**Pull request by antonnystedt**

### CODE REVIEW

File: react/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "plugins": ["react-hooks", "@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react-hooks/recommended"
+  ],
+  "rules": {
```
Comment (albinhubsch) [inline react/.eslintrc.json:12]: can we add `"prettier/prettier": "error"` to the rules section :)?

File: react/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["react-hooks", "@typescript-eslint"],
```
Comment (albinhubsch) [inline react/.eslintrc.json]: add prettier to the plugins array

**Review by albinhubsch — CHANGES_REQUESTED** [review]

File: react/.eslintrc.json (hunk 2 above)
Comment (antonnystedt) [reply react/.eslintrc.json]: Don't know if it makes a difference since we have "plugin:prettier/recommended" in the extends array, but adding it anyway

Comment (albinhubsch) [reply react/.eslintrc.json]: ah, true. merging in, we can always change later as well :)

**Pull request merged**